staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -559,6 +575,193 @@ func TestUpdateNodesInExternalLoadBalancer(t *testing.T) {
 	}
 }
 
+func TestNodeChangesForExternalTrafficPolicyLocalServices(t *testing.T) {
+	node1 := &v1.Node{ObjectMeta: metav1.ObjectMeta{Name: "node0"}, Status: v1.NodeStatus{Conditions: []v1.NodeCondition{{Type: v1.NodeReady, Status: v1.ConditionTrue}}}}
```

Comment:
minor but...the var says node1, the name is node0, which is super confusing with the next one - the name is node1 but the var is node2.  Can we just use the same number in both places?  I don't care if it is not 0-based :)

Reply:
OK, I will make sure to fix this in the PR coming for the 3rd commit. This is recurrent across this test file, so maybe I'll add a dedicated PR for cleaning that up.

Reply:
SGTM